app/views/ht_institutions/show.html.erb

```diff
@@ -5,20 +5,34 @@
 
   <div class="col-sm-6">
     <dl class="dl-horizontal">
-      <dt>ID:</dt> <dd><%= @institution.inst_id %></dd>
-      <dt>GRIN Instance:</dt> <dd><%= @institution.grin_instance %></dd>
+      <dt>Institution ID:</dt> <dd><%= @institution.inst_id %></dd>
+      <dt>Domain:</dt> <dd> <%= @institution.domain %> </dd>
       <dt>Name:</dt> <dd><%= @institution.name %></dd>
+      <dt>Mapped Name:</dt> <dd>
+      <%= @institution.formatted_mapto_name %>
+      <!--      <p><small>Used when displaying the name of a logged-in user's institution, for example in the footer. This is typically used with university system members where individual campuses have separate log-in but the member is the university system as a whole.</small></p> -->
```

Comment:
Might be nice to have this documentation text available; not sure the view is really the right spot for it; not sure how to best format it.

Reply:
I tried it as a Bootstrap tooltip, and it looks OK-ish. Maybe that's for a different iteration.

```
<script>
$(function () {
  $('[data-toggle="tooltip"]').tooltip()
})
</script>
...
<a href="#" data-toggle="tooltip" title="Used when displaying the name of
          a logged-in user's institution, for example in the footer.
          This is typically used with university system members where individual
          campuses have separate log-in but the member is the university system
          as a whole.">
        <i class="glyphicon glyphicon-info-sign"></i>
      </a>
```

Reply:
I tried experimenting with some stuff and just made it worse. Worth re-thinking later how to provide help (both here and for users)